service/src/test/java/bio/terra/pipelines/service/PipelineRunsServiceTest.java

```diff
@@ -296,7 +305,7 @@ void preparePipelineRunImputation() throws MalformedURLException {
 
     Map<String, Map<String, String>> formattedPipelineFileInputs =
         pipelineRunsService.preparePipelineRun(
-            testPipelineWithId, testJobId, testUserId, userPipelineInputs);
+            testPipelineWithId, testJobId, testUserId, userPipelineInputs, testDescription);
```

Comment:
probaby doesnt really matter but testDescription is pointing to the TestUtils.Pipeline_Description which is different from a PipelineRunDescription.  Not sure if we want to make a new static value in TestUtils for pipeline run description or use the pipeline one for both. It might make the tests a tiny bit more clearer and woudl catch if we accidentally were using the pipeline description where we should be using pipeline run description and vice versa

Reply:
ha! yeah I noticed this too and was too lazy to fix it (we were doing it elsewhere before this PR), but that's a good point. I'll fix, thanks!